### DISCUSSION TITLE
Disabling hugo-mods

### DISCUSSION THREAD

**Discussion by hdrz** [kdevo/osprey-delight#41] [Q&A] [answered]
Is it possible to disable hugo-mods and still use this theme (V5 and up) somehow? Currently I'm using V4 and it's great!
There is no way to install go on my computer at the moment....

**Comment by kdevo** [comment, marked as answer]
Hello @hdrz, I'm late with my response, but as of now there is no way to use the newer versions >v5 of the theme without Go installed. Out of curiosity: What makes it impossible for you to install Go?
